src/router/rockets.js

```diff
@@ -0,0 +1,30 @@
+import { useSelector } from 'react-redux';
+import RocketItem from '../components/Rockets/rockets';
+
+const Rockets = () => {
+  const { rockets, isLoading, error } = useSelector((store) => store.rockets);
+  // const dispatch = useDispatch();
+  // useEffect(() => {
+  //   dispatch(getRocketsData());
+  // }, [dispatch]);
```

Comment:
- [ ] Kindly remove all commented pieces of code from your codebase

Reply:
done

Reply:
done